src/main.rs

```diff
@@ -72,19 +70,14 @@ struct Args {
     cmd_config: bool,
 }
 
-fn exit() -> Result<(), anyhow::Error> {
-    stdout().queue(crossterm::cursor::Show)?;
-    crossterm::terminal::disable_raw_mode()?;
-    stdout().flush().map_err(|e| anyhow::anyhow!(e))
-}
 
 fn signal_handler() {
     let signals =
         iterator::Signals::new(vec![SIGTERM, SIGINT, SIGQUIT]).expect("Failed to create Signals");
     for s in signals.forever() {
         match s {
             SIGTERM | SIGINT | SIGQUIT => {
-                if exit().is_err() {
+                if action::interactive::ScopedRaw::restore_terminal().is_err() {
```

Comment:
Should this not always cause an exit code of `130`?

Reply:
My reasoning was to let the user know that something went wrong and `ctrl+c` was not 'successful'. Thinking about it, I doubt that anyone will check the exit code if it's actually `130` or anything else.. So maybe print a warning on error and exit with 130 anyway?